modules/pytket-projectq/pytket/extensions/projectq/backends/projectq_backend.py

```diff
@@ -81,8 +81,35 @@ def _result_id_type(self) -> _ResultIdTuple:
         return (str,)
 
     @property
-    def device(self) -> "Optional[Device]":
-        return None
+    def characterisation(self) -> Dict[str, Any]:
+        return dict()
+
+    @property
+    def backend_info(self) -> BackendInfo:
+        _gate_set = {
+            OpType.SWAP,
```

Comment:
Surely this set should be factored out and used in the place where it's hard-coded below.

Reply:
yes!